test/events.jl

```diff
@@ -0,0 +1,19 @@
+struct TestEvent <: IS.AbstractRecorderEvent
```

Comment:
Forgive this unrelated change. It was easy to fix alongside the addition of `test/components.jl`.